### PR TITLE
make list of all sanitizers hookable

### PR DESCRIPTION
**Pull request by BernhardBaumrock**
Robin created a new sanitizer and I checked if it shows up in $sanitizer->testAll() results, but it didn't. I made a little tweak in his module that only works if we have a hookable getAll() method in the sanitizers class.

See https://processwire.com/talk/topic/21018-sanitizer-transliterate/

### CODE REVIEW

File: wire/core/Sanitizer.php [hunk 1]
```diff
@@ -3567,7 +3567,22 @@ public function maxBytes($value, $maxBytes = 128) {
...
+		foreach($sanitizers as $method) {
+			$results[$method] = $this->$method($value);
+		}
+		return $results;
+	}
+
+	/**
+	 * Get all available sanitizers
+	 *
+	 * @return void
```
Comment (teppokoivula) [inline wire/core/Sanitizer.php]: This isn't quite right – should be `@return array`.

Comment (BernhardBaumrock) [reply wire/core/Sanitizer.php]: thx :)

**Comment by netcarver** [comment]
@BernhardBaumrock As this is an issue, perhaps you could raise an new issue on processwire-issues and link to this PR. I can then mark the issue as containing a suggested fix.

**Comment by BernhardBaumrock** [comment]
Hi @netcarver I've added a comment on this existing issue: https://github.com/processwire/processwire-issues/issues/85 and just saw that it is already closed... Can you reopen it?

**Comment by BernhardBaumrock** [comment]
Yes, thx :)

**Pull request closed without merging**